app/models/project.rb

```diff
@@ -131,4 +130,40 @@ def project_events(order_by = "Completed IS NULL OR Completed = '0000-00-00', Co
   def health_status_available?
     return false
   end
+
+  def self.new_order(person)
+    "case when projects.primary_agent_id = #{person.id} and projects.status_value = 'active'
+          and projects.type = 'Loan' then 12
+
+          when projects.primary_agent_id = #{person.id} and projects.status_value = 'active'
+          and projects.type = 'BasicProject' then 11
+
+          when projects.primary_agent_id = #{person.id} and projects.status_value = 'prospective'
+          and projects.type = 'Loan' then 10
+
+          when projects.primary_agent_id = #{person.id} and projects.status_value = 'prospective'
+          and projects.type = 'BasicProject' then 9
+
+          when projects.primary_agent_id = #{person.id} and projects.type = 'Loan' then 8
+
+          when projects.primary_agent_id = #{person.id} and projects.type = 'BasicProject' then 7
+
+          when projects.secondary_agent_id = #{person.id} and projects.status_value = 'active'
+          and projects.type = 'Loan' then 6
+
+          when projects.secondary_agent_id = #{person.id} and projects.status_value = 'active'
+          and projects.type = 'BasicProject' then 5
+
+          when projects.secondary_agent_id = #{person.id} and projects.status_value = 'prospective'
+          and projects.type = 'Loan' then 4
+
+          when projects.secondary_agent_id = #{person.id} and projects.status_value = 'prospective'
+          and projects.type = 'BasicProject' then 3
+
+          when projects.secondary_agent_id = #{person.id} and projects.type = 'Loan' then 2
+
+          when projects.secondary_agent_id = #{person.id} and projects.type = 'BasicProject' then 1
+
+          else 0 end"
+  end
```

Comment:
Oh also can you please make any SQL keywords (CASE, WHEN, ELSE, AND, END, THEN, etc.) uppercase? It makes things easier to read in my experience.